internal/installutil/aws/ecs.go

```diff
@@ -25,6 +25,7 @@ const (
 	defaultServerTagName     = "waypoint-server"
 	defaultServerTagValue    = "server-component"
 	ServerName               = "waypoint-server"
+	DefaultStaticRunnerName  = "waypoint-static-runner"
```

Comment:
Nit: Should we use `defaultRunnerName('static')` here? Definitely understand if we want to keep the constants simple.

https://github.com/hashicorp/waypoint/blob/main/internal/runnerinstall/runnerinstall.go#L64

Reply:
I think that's a good idea! Hopefully not, but if we change the way runners are named in the future, and thus the default naming convention, it'd be good to already have that in place, instead of the constant.